internal/urlbuilder/urlbuilder_test.go

```diff
@@ -116,7 +116,107 @@ func TestUrlBuilder_buildUrl(t *testing.T) {
 	for i, test := range tests {
 		t.Run("Scenario "+strconv.Itoa(i+1), func(t *testing.T) {
 			builder := UrlBuilder{Path: test.path}
-			url := builder.buildUrl(test.team, test.page, test.perPage, test.filters, test.sort)
+			url := builder.buildUrl(test.team, test.page, test.perPage, test.filters, test.sort, false)
+			assert.Equal(t, test.want, url)
+		})
+	}
+}
+
+func TestUrlBuilder_buildUrl_Preselect(t *testing.T) {
```

Comment:
Just wondering, could you merge this test and the test above, by just adding some "preselect" cases to the above? Or is there a reason you've done it separately?

Reply:
Honestly I just thought it would be simpler to look at/ understand, happy to merge it in if you have a strong preference for it

Reply:
Cool, that makes sense - I suppose it's worth splitting it out rather than making those old tests even longer